eval.py

```diff
@@ -1094,9 +1106,15 @@ def print_maps(all_maps):
             dataset = None        
 
         print('Loading model...', end='')
-        net = Yolact()
-        net.load_weights(args.trained_model)
-        net.eval()
+
+        if deepsparse_available and run_deepsparse:
+            net = DeepsparseWrapper(filepath=args.trained_model, cfg=cfg)
+            global device
+            device = 'cpu'
+        else:
+            net = Yolact()
+            net.load_checkpoint(args.trained_model)
+            net.eval()
         print(' Done.')
 
         if args.cuda:
```

Comment:
do we need to set the global device to cuda here?

Reply:
That helps keep a generic flow, `tensor.to(device)`; Will need if conditions everywhere otherwise

Reply:
Done!